src/apps/btc/btc.h

```diff
@@ -49,6 +49,19 @@ USE_RESULT bool app_btc_xpub(
     char* out,
     size_t out_len);
 
+/**
+ * Returns the identifier xpub.
+ * @param[in] keypath this is a dummy keypath reserved for future use.
```

Comment:
it's not a dummy, it is in use now. pls describe that it must be `m/4541509'/1112098098'`.